crates/re_time_panel/src/paint_ticks.rs

```diff
@@ -68,7 +76,8 @@ fn paint_time_range_ticks(
                 &ui.clip_rect(),
                 time_range, // ns
                 next_grid_tick_magnitude_ns,
-                |ns| Time::from_ns_since_epoch(ns).format_time_compact(),
+                // TODO(paris): Fix 'closures can only be coerced to `fn` types if they do not capture any variables' error.
+                |ns| Time::from_ns_since_epoch(ns).format_time_compact(false),
```

Comment:
You can change `paint_ticks` from taking a `format_tick: fn(i64) -> String` into taking a `format_tick: impl Fn(i64) -> String`

Reply:
Great thanks, updated.